modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/form_builder_field_settings_sidebar.js

```diff
@@ -382,17 +381,14 @@ AUI.add(
 										var previousFieldName = previousSettingsFormFieldContext.fieldName;
 
 										if (!(fieldName === 'type') && fieldName === previousFieldName) {
```

Comment:
Hey @alinedoleron could you group that expressions: `fieldName === previousFieldName` with parentheses?